UnitTest++/Posix/TimeHelpers.cpp

```diff
@@ -27,7 +29,7 @@ namespace UnitTest {
 
    void TimeHelpers::SleepMs(int ms)
    {
-      usleep(static_cast<useconds_t>(ms * 1000));
+      std::this_thread::sleep_for(std::chrono::microseconds(ms * 1000));
```

Comment:
We can't use `<chrono>` in UnitTest++ at this time. UnitTest++ does not require a modern compiler.